cetmix_tower_server/models/cx_tower_variable_option.py

```diff
@@ -0,0 +1,29 @@
+# Copyright (C) 2022 Cetmix OÜ
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+
+from odoo import fields, models
+
+
+class TowerVariableOption(models.Model):
+    _name = "cx.tower.variable.option"
+    _description = "Cetmix Tower Variable Options"
+    _order = "priority, name"
+
+    name = fields.Char(string="Option Value", required=True)
+    variable_id = fields.Many2one(
+        comodel_name="cx.tower.variable",
+        string="Variable",
+        required=True,
+        ondelete="cascade",
+        index=True,
+    )
+    priority = fields.Integer(string="Priority", default=0)
```

Comment:
`0` is default value for Integer fields. So either do not add this key at all, or use some different value. Eg like it's done [here](https://github.com/cetmix/cetmix-tower/blob/1b9121272c56eb300f3a65704ddcbf6572c76df3/cetmix_tower_server/models/cx_tower_plan_line.py#L20) or [here](https://github.com/odoo/odoo/blob/cc0060e889603eb2e47fa44a8a22a70d7d784185/addons/account/models/account_move.py#L3316).

I think it's also better to rename this field to `sequence` too, because this name is more relevant for it.

Reply:
Ok, I will change